west.yml

```diff
@@ -14,7 +14,10 @@ manifest:
     - name: zephyr
       remote: zephyrproject-rtos
       revision: zephyr-v3.1.0
-      import: true
+      import:
```

Comment:
:raised_hands: love it

Reply:
actually we should probably document this in the README. I think this might break if you try to target some real target.

Reply:
Ah, It does indeed -- I didn't realize you were targeting other stuff beyond qemu

Reply:
yeah the default example is like `west build -b nrf9160dk_nrf9160_ns -s zephyr-memfault-example/app`... i like using it for quick and dirty on stm32f4's and such too

Reply:
ok -- I'll move this to draft -- there's some way to chain manifest iirc, might look into that approach instead to have a super stripped down qemu variant